File: StarkAid.Api/DTOs/DisparoResponse.cs
namespace StarkAid.Api.Dtos
{
    public class DisparoResponse
    {
        public Guid Id { get; set; }
        public Guid DispositivoId { get; set; }
        public string DispositivoNome { get; set; } = string.Empty;
        public DateTimeOffset DisparadoEm { get; set; }
        public string Mensagem { get; set; } = string.Empty;
        public bool Confirmado { get; set; }
        public DateTimeOffset? ConfirmadoEm { get; set; }
    }
}

File: starkaidautomacao/app/src/main/java/com/starkaid/starkaidapp/models/DisparoResponse.kt
package com.starkaid.starkaidapp.models

data class DisparoResponse(
    val id: String,
    val dispositivoId: String,
    val dispositivoNome: String,
    val disparadoEm: String,
    val mensagem: String,
    val confirmado: Boolean,
    val confirmadoEm: String?
)